[IMP] accounting: clarify invoice sequence format (#12053)
odoo/documentation: content/applications/finance/accounting/customer_invoices.rst

.. _accounting/invoice/sequence:

Invoice sequence
================

When confirming an invoice, Odoo generates a unique invoice reference number. By default, Odoo uses
the following sequence format `INV/year/incrementing-number` (e.g., `INV/2025/00001`), which
restarts from `00001` each year.

However, it is possible to :ref:`change the sequence format <accounting/invoice/resequencing>` and
its periodicity, and to :ref:`mass-resequence invoices <accounting/invoice/mass-resequencing>`.

.. note::
   Changes made to reference numbers are logged in the chatter.

.. _accounting/invoice/resequencing:

Changing the default sequence
-----------------------------

To customize the default sequence, open the last confirmed invoice, click :guilabel:`Reset to
Draft`, and edit the invoice's reference number.

.. image:: customer_invoices/reference-number.png
   :alt: Editing the reference number of an invoice.

Odoo then explains how the detected format will be applied to all future invoices. For example, if

> you add the current invoice's month, the sequence's periodicity will change to every month instead

Reviewer: To keep the same style as the rest of the page.
```suggestion
the current invoice's month is added, the sequence's periodicity will change to every month instead
```

of every year.

.. image:: customer_invoices/sequence-dialog.png
   :alt: Editing the reference number of an invoice.

.. tip::

>    You can edit the sequence format directly when creating the first invoice of a given sequence
>    period.

Reviewer: Same comment as above.
```suggestion
   The sequence format can be edited directly when creating the first invoice of a given sequence
   period.
```
